source/elements/oneMKL/source/domains/spblas/data_types/create_dense_vector.rst

```diff
@@ -0,0 +1,162 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_create_dense_vector:
+
+create_dense_vector
+===================
+
+Initializes a ``dense_vector_handle_t`` object with the provided data.
+
+.. rubric:: Description and Assumptions
+
+The ``oneapi::mkl::sparse::create_dense_vector`` function initializes the
+``dense_vector_handle_t`` object with the provided data.
+
+In the case of buffers, the reference count of the provided buffer is
+incremented which extends the lifetime of the underlying buffer until the
+``dvhandle`` is destroyed with ``destroy_dense_vector`` or the data is reset
+with ``set_dense_vector_data``.
+
+In the case of USM, the object does not take ownership of the data.
+
+See :ref:`onemkl_sparse_dense_vector_handle`.
+
+.. _onemkl_sparse_create_dense_vector_buffer:
+
+create_dense_vector (Buffer version)
+------------------------------------
+
+.. rubric:: Syntax
+
+.. code-block:: cpp
+
+   namespace oneapi::mkl::sparse {
+
+       template <typename dataType>
+       void create_dense_vector (sycl::queue                                &queue,
+                                 oneapi::mkl::sparse::dense_vector_handle_t *p_dvhandle,
+                                 std::int64_t                               size,
+                                 sycl::buffer<dataType, 1>                  &val);
+
+   }
+
+.. container:: section
+
+   .. rubric:: Template parameters
+
+   dataType
+      See :ref:`supported template types<onemkl_sparse_supported_types>`.
+
+.. container:: section
+
+   .. rubric:: Input parameters
+
+   queue
+      The SYCL command queue which will be used for SYCL kernels execution.
+
+   p_dvhandle
+      The address of the ``p_dvhandle`` object to be initialized. Must only be
+      called on an uninitialized ``dense_vector_handle_t`` object.
+
+   size
+      Number of elements of the provided data ``val``. Must be at least 0.
+
+   val
+      Buffer of length ``size``. Holds the data to initialize ``p_dvhandle``
+      with.
```

Comment:
```suggestion
      Buffer of length at least ``size``. Holds the data to initialize ``p_dvhandle``
      with.
```

Reply:
Addressed in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/13dd2e5721d52c730695e51fe3b5471580f0d75e